src/main/kotlin/translations/TranslationFiles.kt

```diff
@@ -223,6 +260,17 @@ object TranslationFiles {
         doc.insertString(rootObject.lastChild.prevSibling.textOffset, content)
     }
 
+    private fun PsiFile.findKeyForTextAsJson(text: String): String? {
+        val rootObject = this.firstChild as? JsonObject ?: return null
+        return rootObject.propertyList.firstOrNull {
+            if (it.value is JsonStringLiteral) {
+                (it.value as JsonStringLiteral).value == text
+            } else {
+                false
+            }
```

Comment:
Can be `(it.value as? JsonStringLiteral)?.value == text`

Reply:
[271746a](https://github.com/minecraft-dev/MinecraftDev/pull/2292/commits/271746a3b0fea06c09f033fef527351451d25505)